docs/dev/README.md

```diff
@@ -19,6 +19,15 @@ https://github.com/triplea-game/triplea/tree/master/eclipse/format
 Useful:
   - [EGit in Eclipse](http://www.eclipse.org/egit/) - with [tutorial](http://www.vogella.com/tutorials/EclipseGit/article.html)
 
+Lombok:
+  Install Plugin ([reference](https://projectlombok.org/downloads/lombok.jar))
```

Comment:
* Devs should be directed to install the same version of Lombok listed in the Gradle dependencies to ensure consistency between Gradle and IDE builds.
* Probably need a blurb that says devs must re-install the Lombok IDE plugin any time we upgrade Lombok in Gradle.

Reply:
Reasonable enough - updated in: b0473e7

I noticed the formatting was off in the rendered markdown and fixed that as well.